Source/WebCore/platform/graphics/cocoa/IOSurface.h

```diff
@@ -123,8 +115,15 @@ class IOSurface final {
         }
 
     private:
+        Locker(IOSurfaceRef surface, uint32_t flags)
+            : m_surface(surface)
```

Comment:
It's a bit surprising that this class doesn't retain the surface.

Reply:
Yeah. It can go either way. I suppose having a ref is more in line with the expectations. Added.
(It's a bit unfortunate that storing Locker duplicates the storage for the surface.)